go/pkg/feed/service.go

```diff
@@ -25,6 +26,7 @@ type Config struct {
 	Logger    *zap.Logger
 	IndexerDB *gorm.DB
 	PinataJWT string
+	Networks  networks.NetworkStore
```

Comment:
if the value is NetworkStore then maybe naming attr name: netstore is less confusing ?

Reply:
since it's acting like the frontend "networks" package, I though I could name it simpler, not a strong opinion